lemur/plugins/lemur_digicert/plugin.py

```diff
@@ -317,10 +319,71 @@ def __init__(self, *args, **kwargs):
 
         self.session.hooks = dict(response=log_status_code)
 
+        # max_retries applies only to failed DNS lookups, socket connections and connection timeouts,
+        # never to requests where data has made it to the server.
+        # we Retry we also covers HTTP status code 406, 500, 502, 503, 504
+        retry_strategy = Retry(total=3, backoff_factor=0.1, status_forcelist=[406, 500, 502, 503, 504])
+        adapter = requests.adapters.HTTPAdapter(max_retries=retry_strategy)
+        self.session.mount("https://", adapter)
+
         super().__init__(*args, **kwargs)
 
-    def get_certificates(self):
-        pass
+    def get_certificates(self, options, **kwargs):
+        """Fetch all Digicert certificates."""
+        base_url = current_app.config.get("DIGICERT_URL")
+
+        # make request
+        search_url = f"{base_url}/services/v2/order/certificate"
+
+        certs = []
+        offset = 0
```

Comment:
Can we use len(certs) instead of offset? I'm concerned that the API may return less than the limit. In this case, the source plugin would skip over some certs